src/commands/init.ts

```diff
@@ -134,20 +114,79 @@ export const validatePersonalAccessToken = async (
   }
 };
 
+export const isIntrospectionAzureDefined = (curConfig: ConfigYaml): boolean => {
+  if (!curConfig.introspection) {
+    return false;
+  }
+  const intro = curConfig.introspection;
+  return intro.azure !== undefined;
+};
+
+export const handleIntrospectionInteractive = async (
+  curConfig: ConfigYaml
+): Promise<void> => {
+  if (!isIntrospectionAzureDefined(curConfig)) {
+    curConfig.introspection = {
+      azure: {
+        key: new Promise(resolve => {
+          resolve(undefined);
+        })
+      }
+    };
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const azure = curConfig.introspection!.azure!;
+
+  const ans = await inquirer.prompt([
+    promptBuilder.azureStorageAccountName(azure.account_name),
+    promptBuilder.azureStorageTableName(azure.table_name),
+    promptBuilder.azureStoragePartitionKey(azure.partition_key),
+    promptBuilder.azureStorageRepoAccessKey(azure.source_repo_access_token),
```

Comment:
`azureStorageRepoAccessKey` should not be associated with `azure.source_repo_access_token`? Those are two different fields

Reply:
accept
I will rename to helper function to avoid confusion. `azureStorageRepoAccessKey` is just a helper function. 
one is helper function and another is configuration variable.